.circleci/config.yml

```diff
@@ -46,7 +46,6 @@ jobs:
       - run: pip install --user 'tox<5'
       - run: tox -e pre-commit
       - run: tox -e migrations
-      - run: tox -e lint
```

Comment:
I think we still want this in tox? What happens if a user doesn't run lint locally?

Reply:
Ah, ok 👍

Reply:
pre-commit is super clever, it maintains a virtual env for each linting tool. So in this case, it's an improvement over `tox -e lint` since we don't want a virtualenv for _all_ linting, but one for each tool (so they don't conflict).